vllm/core/block_manager_v1.py

```diff
@@ -255,12 +255,26 @@ def __init__(
                 Device.CPU, block_size, num_cpu_blocks)
         # Mapping: seq_id -> BlockTable.
         self.block_tables: Dict[int, BlockTable] = {}
+        # Mapping: req_id -> BlockTable
+        # Note that each SequenceGroup has a unique
+        # request ID
+        self.cross_block_tables: Dict[str, BlockTable] = {}
+
+    def get_seq_num_required_blocks(self, seq: Sequence) -> int:
```

Comment:
It's a matter of personal preference, but it's a good practice to prefix internal methods with `_` to differentiate from methods that are called by the client code.

Reply:
Thanks! Just pushed a commit to fix this